### ISSUE TITLE
eDEX slows down heavily when requests to ipapi fails

### ISSUE THREAD

**Issue by GitSquared** [GitSquared/edex-ui#270]
The [IPapi](https://github.com/GitSquared/ipapi) service I built for eDEX got absolutely wrecked by the fame and regularly crashes with weird 503 errors (this runs on freakin AWS, through Now.)

The instances are getting replaced when they fail but i noticed this issue in the meantime. (current scaling is 2 instances in BRU & 2 in SFO).

Ho, by the way, you madlads used up 41.25 GB of bandwith in 3 days. On a API that outputs a bit of JSON. I'll probably need to rewrite it.

**Comment by gggauravgandhi** [comment]
Try this, `curl ipinfo.io`. If we can use that service so we can reduce over overhead for scaling for now.

**Comment by GitSquared** [comment]
@gggauravgandhi Yeah i don't want to pay $49.00/month for a service i can host for less than 10$ on Now. Also not too keen on using external services, for privacy related reasons (i know for a fact that ipinfo.now.sh doesnt log anything, since i wrote it)

**Comment by gggauravgandhi** [comment]
@GitSquared , Curl request works without any API key, thus without any registration or payment. For other services, we have to pay. Also, ipinfo.io is not a small startup, its well-established company, so we can trust at a certain level as much we can trust API running on some developers EC2 machine(No offence, just stating facts).

**Comment by GitSquared** [comment]
@gggauravgandhi My API is open-source, that's the difference.
